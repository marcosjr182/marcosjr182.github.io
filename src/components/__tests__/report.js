import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Report from '../report';

const report_info = {
  description: 'ReportDescription',
  average_of_tests: 10
}

const wrapper = shallow(<Report report={report_info} />);

describe('Report component', () => {
  it('should render info correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})
