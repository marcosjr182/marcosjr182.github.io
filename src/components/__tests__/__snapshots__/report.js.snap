exports[`Report component should render info correctly 1`] = `
<div
  className="col-xs-6 col-sm-4 card-container">
  <div
    className="col-xs-12 card report">
    <ReportInfo
      name="description"
      value="ReportDescription" />
    <ReportInfo
      name="average_of_tests"
      value={10} />
  </div>
</div>
`;
